.github/workflows/reusable-docker-build.yml

```diff
@@ -70,10 +76,8 @@ jobs:
           file: containerfiles/Dockerfile
           build-args: |
             TARGETBINARY=${{ inputs.target-binary }}
-          platforms: 'linux/amd64,linux/arm64'
-          push: true
+          platforms: "linux/amd64,linux/arm64"
+          push: ${{ env.LOGIN_DOCKER || 'false' }}
```

Comment:
Do fork PRs get pushed to the registries today? I wasn't sure about that one. If not, I think we could update this to check if the job is from a fork and not push.

```yaml
push: ${{ github.event.pull_request.head.repo.full_name == 'org/repo' }}
```

Reply:
They don't, I updated to validating that it's owned by astriaorg